tests/scripts/diff-remove-spicy-location

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+#
+# Replace (a/b/c.spicy:1:2-3:4) with (a/b/c.spicy:<location>)
+
+sed -E 's/\((.+)\.spicy:[0-9]+:[0-9]+(-[0-9]+:[0-9]+)?\)/(\1.spicy:<location>)/g'
```

Comment:
Nice, this supports both positions `col:line` and ranges `col1:line1-col2:line2`, even though only positions are used here.